har_dataset.py

```diff
@@ -773,12 +836,22 @@ def __prepare_sisfall__(path, split):
     print(f'Parsing {len(all_files)} files...')
     recordings = []
     for i, file in enumerate(all_files):
-        class_name, subject, recording = file.stem.split('_')
-        subject = file.parent.name # override possible labeling error in SA15/D17_SE15...
-        # cls = int(class_name[1:]) # infer class at the end because of skipped activities
-        # subject = int(subject[2:])
-        #TODO: map subject name to number == identity label 
-        recording = int(recording[1:])
+        if half_data==True:
+            if subject not in ids[split]:
```

Comment:
Again, subject called before assigned. 

Else not needed, same procudre as in motionsense should be enough if you only split by subjects